tests/unit/test_query.py

```diff
@@ -169,8 +169,10 @@ def test_from_api_repr(self):
         self.assertEqual(field._type, "STRING")
 
     def test_to_api_repr(self):
-        from google.cloud.bigquery.query import ScalarQueryParameterType
-        from google.cloud.bigquery.query import StructQueryParameterType
+        from google.cloud.bigquery.query import (
+            ScalarQueryParameterType,
+            StructQueryParameterType,
+        )
```

Comment:
Why do we do import within each test instead of at the top of the test module?

Reply:
And i believe these import changes are from running `nox -s blacken` and `nox -s lint`.